docs/3.x.x/en/advanced/customize-admin.md

```diff
@@ -111,90 +111,119 @@ This will replace the folder's content located at `./admin/admin/build`. Visit h
+The administration URL will be https://yourfrontend.com/dashboard and every request from the panel will hit the backend at https://yourbackend.com. The plugins will be injected through the `origin` (means the API itself). In other words, the plugins URLs will be `https://yourbackend.com/admin/content-manager/main.js`.
+
+> Note: How it is possible? The API (the Strapi server) owns the plugin and these plugins are exposed through `https://yourbackend.com/admin/**/main.js`
+
+
+The DOM should look like this:
 
 **Path —** `./admin/admin/build/index.html`.
 ```html
 <html>
   <head></head>
   <body>
     <div id="app"></div>
-    <script type="text/javascript" src="https://admin.myapp.com/admin/vendor.dll.js"></script>
-    <script type="text/javascript" src="https://admin.myapp.com/admin/main.js"></script>
-    <script src="https://api.myapp.com:8080/admin/content-manager/main.js"></script>
-    <script src="https://api.myapp.com:8080/admin/settings-manager/main.js"></script>
-    <script src="https://api.myapp.com:8080/admin/content-type-builder/main.js"></script>
+    <script type="text/javascript" src="/dashboard/vendor.dll.js"></script>
+    <script type="text/javascript" src="/dashboard/main.js"></script>
+    <script src="https://yourbackend.com/admin/content-manager/main.js"></script>
+    <script src="https://yourbackend.com/admin/settings-manager/main.js"></script>
+    <script src="https://yourbackend.com/admin/content-type-builder/main.js"></script>
   </body>
 </html>
 ```
 
 > Note: The plugins are injected using the `./admin/admin/build/config/plugins.json`. To see the plugins URLs in the `index.html`, you need to launch the administration panel in the browser.
 
-#### With the plugins on another server
+#### Deploy the administration panel and the plugins on another server than the API
 
 In this case, we suppose that you decided to put your administration and the plugins on the same server but on a different server as the API.
 
 **Path —** `./config/environment/**/server.json`.
-```json
+```js
 {
+  "host": "localhost",
+  "port": 1337,
+  "autoReload": {
+    "enabled": false
+  },
+  "cron": {
+    "enabled": false
+  },
   "admin": {
     "build": {
-      "host": "https://admin.myapp.com",
-      "backend": "https://api.myapp.com:8080",
+      "host": "https://yourfrontend.com/dashboard", // Note: The custom path has moved directly in the host URL.
+      "backend": "https://yourbackend.com",
       "plugins": {
-        "source":  "host",
-        "folder": "plugins"
+        "source":  "host" // What does it means? The script tags in the index.html will use the host value to load the plugins (ex: https://yourfrontend.com/dashboard/plugins/content-manager/main.js).
```

Comment:
Missing coma after `"source":  "host"` ..

Reply:
!! Warning !!
JSON format doesn't accept comment ! 
Did you ever try start once?

Reply:
Sorry.. I miss that it's docs.. 👍

Reply:
@edimitchel Thank you! It's fixed 👍